.gitignore

```diff
@@ -5,4 +5,4 @@ azureenv.sh
 .vscode/
 !terraform-provider-akc_v*.exe
 .publish/
-vendor/
\ No newline at end of file
+vendor/
```

Comment:
Do we really need to checkin the `vendor` folder ?

Reply:
I added it to .gitignore, so it won't be checked in my mistake, so I am already in agreement with you 👍🏻

Reply:
🤦